src/Core/Framework/DataAbstractionLayer/Event/DefinitionValidatorFilterEvent.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopware\Core\Framework\DataAbstractionLayer\Event;
+
+use Shopware\Core\Framework\DataAbstractionLayer\EntityDefinition;
+
+class DefinitionValidatorFilterEvent
```

Comment:
When (if) you refactor would be nice if you remove the methods and just make violations public. You could keep the filter method if you wanted.